### ISSUE TITLE
support running in subinterpreters

### ISSUE THREAD

**Issue by graingert** [pytest-dev/pytest-xdist#474]

**Comment by RonnyPfannschmidt** [comment]
currently that would be an issue for execnet, a xdist doesn't have a own mechanism

**Comment by RonnyPfannschmidt** [comment]
FYI Execnet is currently unmaintained

**Comment by graingert** [comment]
Might it be worth merging it into xdist?

**Comment by RonnyPfannschmidt** [comment]
nope, we already have a issue about using multiprocessing and mitogen, execnet is "dead" and i stepped down as the maintainer because other projects are farther along and its simply not sensible to maintain it longer

**Comment by nicoddemus** [comment]
The way forward would be to actually release a new project, say `pytest-xdist2`, which uses diferent backends to execute tests in the same machine.

A new project because it would drop ssh support, because the user needs to take care of configuring dependencies on the host themselves, which limits its usefulness.

**Comment by nicoddemus** [comment]
Hmmm actually mitogen seems all about executing things remotely.
